timesketch/frontend-ng/src/components/Explore/EventDetail.vue

```diff
@@ -43,8 +56,62 @@ limitations under the License.
                         :xmlString="value"
                       ></ts-format-xml-string>
                     </v-dialog>
+                    <v-menu
+                      v-if="checkContextLinkDisplay(key, value)"
+                      offset-y
+                      transition="slide-y-transition"
+                    >
+                      <template v-slot:activator="{ on, attrs }">
+                        <v-btn
+                          icon
+                          color="primary"
+                          x-small
+                          style="cursor: pointer"
+                          v-bind="attrs"
+                          v-on="on"
+                        >
+                          <v-tooltip top close-delay=300 :open-on-click="false">
+                            <template v-slot:activator="{ on }">
+                              <v-icon
+                                v-on="on"
+                                small
+                              >
+                                mdi-open-in-new
+                              </v-icon>
+                            </template>
+                            <span>Context Lookup</span>
+                          </v-tooltip>
+                        </v-btn>
+                      </template>
+                      <v-list dense>
+                        <v-list-item
+                          v-for="(item, index) in getContextLinkItems(key)"
+                          :key="index"
+                          style="cursor: pointer"
+                          @click.stop="contextLinkRedirect(key,item,value)"
+                        >
+                          <v-list-item-title v-if="getContextLinkRedirectState(key,item)" >{{ item }}</v-list-item-title>
+                          <v-list-item-title v-else >{{ item }}*</v-list-item-title>
```

Comment:
Maybe replace the start with. "(internal)"?

Reply:
Since I leave the configuration for the redirect warning to each admin, the linked service can also be external and show now redirect warning (e.g. if they don't care about checking hashes in VT). In this situation it would show `Virustotal (internal)` which is not correct.
Hence the idea to go with just a marker for "no redirect warning", which can be improved.

For now I would like to keep the `*` for this PR and take the topic in my next discussion with UX.

Reply:
SG